admin/onpage/class-onpage-request.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * This class will fetch a new status from OnPage.org and if it's necessary it will notify the site admin by email and
+ * remove the current meta value for hidding the notice for all admin users
+ */
+class WPSEO_OnPage_Request {
+
+	/**
+	 * @var array The array that is return by the wp_remote_get request
+	 */
+	private $response;
+
+	/**
+	 * Setting the response by doing the request to given target_url
+	 *
+	 * @param string $target_url The target url for requesting the status.
+	 */
+	public function __construct( $target_url ) {
+		$this->response = $this->do_request( $target_url );
+	}
+
+	/**
+	 * Returns the fetched response
+	 *
+	 * @return array
+	 */
+	public function get_response() {
+		return $this->response;
+	}
+
+	/**
+	 * Doing the remote get and returns the body
+	 *
+	 * @param string $home_url The home url.
+	 *
+	 * @return array
+	 */
+	protected function get_remote( $home_url ) {
```

Comment:
This parameter has nothing to do with the `home_url()`, so just like it is in the constructor this should be `$target_url`.